master/internal/db/postgres_checkpoints.go

```diff
@@ -4,8 +4,36 @@ import (
 	"fmt"
 
 	"github.com/google/uuid"
+	"github.com/pkg/errors"
+
+	"github.com/determined-ai/determined/master/pkg/model"
 )
 
+// CheckpointByUUID looks up a checkpoint by UUID, returning nil if none exists.
+func (db *PgDB) CheckpointByUUID(id uuid.UUID) (*model.Checkpoint, error) {
+	var checkpoint model.Checkpoint
+	if err := db.query(`
+SELECT *
+FROM checkpoints_view c
+WHERE c.uuid = $1`, &checkpoint, id.String()); errors.Cause(err) == ErrNotFound {
+		return nil, nil
+	} else if err != nil {
+		return nil, errors.Wrapf(err, "error querying for checkpoint (%v)", id.String())
+	}
+	return &checkpoint, nil
+}
+
+// CheckpointByUUIDs looks up a checkpoint by list of UUIDS, returning nil if error.
+func (db *PgDB) CheckpointByUUIDs(ckptUUIDs []uuid.UUID) ([]model.Checkpoint, error) {
+	var checkpoints []model.Checkpoint
+	if err := db.queryRows(`
+	SELECT * FROM checkpoints_view c WHERE c.uuid 
```

Comment:
Can you use consist formatting between this and the above query (how the query is formatted)?